common/config/src/argv.rs

```diff
@@ -121,6 +121,13 @@ pub struct ArgumentOptions {
     #[structopt(long, env = env_vars::LOG_K8S_EVENTS)]
     log_k8s_events: Option<K8sTrackingConf>,
 
+    /// Determines whether the agent should produce metrics-servers usage logs.
+    /// When set this agent will poll the metrics-server on the cluster to get usage statistics
+    /// combine them with other pod/node statistics for enhancements to the web application.
+    /// Defaults to "off".
+    #[structopt(long, env = env_vars::LOG_METRIC_SERVER_STATS)]
+    log_metric_server_stats: Option<K8sTrackingConf>,
```

Comment:
Does having the use the `Options<K8sTrackingConf>` make this dependent on the other two feature that using this option type?

Reply:
Yeah, I'm not sure if it was meant to be general purpose or if the other two configurations needed to be coupled and that's why it was written that way.